pkg/scheduler/apis/config/v1alpha1/defaults.go

```diff
@@ -30,6 +31,19 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
 
+var defaultTopologySpreadConstraints = []corev1.TopologySpreadConstraint{
+	{
+		MaxSkew:           1,
+		TopologyKey:       corev1.LabelHostname,
+		WhenUnsatisfiable: corev1.ScheduleAnyway,
+	},
+	{
+		MaxSkew:           1,
+		TopologyKey:       corev1.LabelZoneFailureDomain,
```

Comment:
I would expect to use the GA label, not the beta one

Reply:
Updated.